lib/rubocop/cop/rspec/missing_example_group_argument.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks that the first argument to an example group is not empty.
+      #
+      # @example
+      #   # bad
+      #   describe do
+      #   end
+      #
+      #   RSpec.describe do
+      #   end
+      #
+      #   # good
+      #   describe TestedClass do
+      #   end
+      #
+      #   describe "A feature example" do
+      #   end
+      class MissingExampleGroupArgument < Cop
+        MSG = 'The first argument to `%<method>s` should not be empty.'.freeze
+
+        def_node_matcher :example_group_send?, ExampleGroups::ALL.send_pattern
```

Comment:
We don't specify if it's send or block or whatever in matcher names. `example_group?` reads better.

Reply:
Also if you prefer to reuse the existing definition of `example_group?` you can use it, change the cop to listen to `on_block` and then return on `node.send_node.arguments?`

Reply:
Good idea. Thanks.